debian/syslog-ng.conf.example

```diff
@@ -38,19 +38,19 @@ destination syslog { file("/var/log/syslog.log"); };
 destination user { file("/var/log/user.log"); };
 destination uucp { file("/var/log/uucp.log"); };
 
-# This files are the log come from the mail subsystem.
+# These files are the logs that come from the mail subsystem.
 #
 destination mailinfo { file("/var/log/mail/mail.info"); };
 destination mailwarn { file("/var/log/mail/mail.warn"); };
 destination mailerr { file("/var/log/mail/mail.err"); };
 
-# Logging for INN news system
+# Logging for INN news system.
 #
 destination newscrit { file("/var/log/news/news.crit"); };
 destination newserr { file("/var/log/news/news.err"); };
 destination newsnotice { file("/var/log/news/news.notice"); };
 
-# Some `catch-all' logfiles.
+# Some `catch-all` logfiles.
```

Comment:
This is a text file and not markdown syntax. In our documentation, we use the convention shown above, so I would leave this one.